idunn/geocoder/nlu_client.py

```diff
@@ -237,9 +242,13 @@ async def get_intentions(self, text, lang, focus=None):
             {"brand_query": brand_query, "cat_query": cat_query, "place_query": place_query}
         )
 
-        # If there is a label for both a category and a brand, we consider that the request is
-        # ambiguous and ignore both.
-        if bool(brand_query) ^ bool(cat_query):
+        try:
+            if brand_query and cat_query:
+                raise NluClientException("detected a category and a brand")
+
+            if not brand_query and not cat_query:
+                raise NluClientException("no category or brand detected")
```

Comment:
This is a very common case, for cities, addresses etc.  
Are we sure that we want to include it in logs ?

Reply:
I'm not exactly sure, this is indeed not an unexpected case (so info log level sounds fine) but it may be useful to extract some dataset out of the logs? I wouldn't making it more quiet if you think it is better to keep more concise logs :+1: